nifi-operator/tests/scale_cluster.rs

```diff
@@ -1,6 +1,7 @@
 pub mod common;
 
 use crate::common::nifi::maximize_client_verification_time_out;
+use crate::common::zookeeper::build_zk_test_cluster;
```

Comment:
```suggestion
use common::nifi::maximize_client_verification_time_out;
use common::zookeeper::build_zk_test_cluster;
```

If you start with being nitpicking, you cannot stop in between. I wish I had not started. :D